civis/parallel.py

```diff
@@ -409,7 +409,7 @@ class JobSubmissionError(Exception):
     pass
 
 
-def _robust_result_download(output_file_id, client, n_retries=5, delay=0.0):
+def _robust_pickle_download(output_file_id, client, n_retries=5, delay=0.0):
```

Comment:
Did we want a bigger doc string here? The other function has one which is why I am asking.